styles.css

```diff
@@ -1246,7 +1248,7 @@
   }
 
   .proj-desc {
```

Comment:
- [  ] [OPTIONAL] I have noticed that the `project description` is overlapping the subtitle 😞 . Kindly ensure that the description is independent.

Your design 😎 
![overlapping](https://user-images.githubusercontent.com/99394374/213866110-2364239c-fa0b-43d9-837e-758175380520.PNG)